app/editor/src/features/content/form/ContentTranscriptForm.tsx

```diff
@@ -48,6 +48,7 @@ export const ContentTranscriptForm: React.FC = () => {
       <FormikTextArea
         name="transcription"
         label="Transcript"
+        style={{ height: '290px', minHeight: '65px' }}
```

Comment:
@CelloLiu don't do inline styles.
@jdtoombs This won't expand or contract will it?

Reply:
@Fosol - `resize: none` needs to be applied to disable the functionality of a user changing the height/width

Reply:
Updated, thanks